lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -314,6 +314,16 @@ def setup_purchase(money, options = {})
         commit(:post, 'payment_intents', post, options)
       end
 
+      def inquire(intent_id, options = {})
+        if intent_id.start_with?('pi_')
+          show_intent(intent_id, options)
+        elsif intent_id.start_with?('seti_')
+          retrieve_setup_intent(intent_id, options)
+        else
+          unsupported_inquire_response
```

Comment:
This is not needed since it wouldn't be possible for a transactions to have a different intent_id. Only PaymentIntent and SetupIntent are supported by StripePI